src/antsibull_docs/schemas/collection_config.py

```diff
@@ -12,6 +12,11 @@
 from antsibull_docs._pydantic_compat import v1 as p
 
 
+class ChangelogConfig(p.BaseModel):
+    # Whether to write the changelog
+    write_changelog: bool = False
```

Comment:
Do we think we might ever default this to true? maybe in a new major version? 
I guess we'd want to discuss in the forum and/or dawgs meeting.

If so, we could announce the future default value in the changelog in this PR too

Reply:
It should never default to true, but it could default to something like "auto" which checks whether changelogs/changelog.yaml is present and only then sets it to true. But that would have to wait for a new major release.

Also I don't think the default should switch; I'd rather have a configurable toggle for the docsite build which allows to override this for collections.